### PR TITLE
Reorganize code

### PR DESCRIPTION
**Pull request by rhcarvalho**
This moves code away from tasks.go to more specific files.

It should make it easier to find all code related to a certain task type.

### CODE REVIEW

File: nagios.go [hunk 1]
```diff
@@ -26,3 +49,22 @@ func GetNagiosHistoricalData(r Runner, project, pod string) Task {
 		return r.Run(cmd, path)
 	}
 }
+
+// getResourceNamesBySubstr returns a list of names for the provided resource type that contain
+// the provided string, in the provided project.
+func getResourceNamesBySubstr(project, resource, substr string) ([]string, error) {
```
Comment (philbrookes) [inline nagios.go]: not sure this should be here, it will be needed by the millicore tasks too.

Comment (rhcarvalho) [inline nagios.go]: Hmmm, for sure it had nothing to do with logs. Where should it go then?

Comment (philbrookes) [inline nagios.go]: maybe we need a new file for shared functions and that? Like `shared.go` or something?

**Comment by philbrookes** [comment]
:+1: LGTM

File: nagios.go (hunk 1 above)
Comment (rhcarvalho) [inline nagios.go]: Talked on IRC, staying here for now, may move somewhere else in the future when it will have more callers.

**Pull request merged**